fuzz-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+fuzz_list_cmd="cargo fuzz list"
+
+$fuzz_list_cmd | while read -r target; do  
+  echo "----- fuzzing $target ------"
+  target_cmd="cargo fuzz run $target -timeout=300"  
```

Comment:
This command is not working. You should add `--` before the `-timeout` parameter.